### PR TITLE
Update ledger intergration

### PR DESCRIPTION
**Pull request by TriPhan96**
- Create account/subaccount and login with Ledger Nano S devices
- Sign simple and aggregate transaction
- Work with Symbol BOLOS app v0.0.4 or higher

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "start": "electron .",
         "test": "vue-cli-service test:unit",
         "dev": "vue-cli-service serve ",
-        "build": "npm run build:web",
+        "build": "npm run electron-rebuild && npm run build:web",
```
Comment (bassemmagdy) [inline package.json:15]: @TriPhan96 that breaks releasing wallet locally

Comment (tri-vo-fds) [reply package.json:15]: Ok @bassemmagdy. I will change it to:

Comment (TriPhan96) [reply package.json]: Hi @bassemmagdy We have updated to the code applied @tri-vo-fds recommendation. Please have a look at it. Thanks

Comment (bassemmagdy) [reply package.json]: @tri-vo-fds @TriPhan96 still breaking because of `npm run electron-rebuild` statement

Comment (TriPhan96) [reply package.json]: @bassemmagdy Could you please screenshot the error you are encountered when running the release command on your machine? Maybe the node version you are using would help us to reproduce your issue with this PR

Comment (TriPhan96) [reply package.json]: @bassemmagdy We have update the node-hid package to make the release wallet locally, please test it again, thank you.

Comment (bassemmagdy) [reply package.json]: @TriPhan96 it worked for releasing wallet locally only for mac, but for windows and linux I always get this [output](https://drive.google.com/file/d/1UhXbaK5SxtmhWl2sManyUDP4jbX21tH3/view?usp=sharing), I'm using Osx with node v12.18.3. Also, there are some conflicts that need to be resolved.

**Comment by bassemmagdy** [comment]
closed due to another [PR](https://github.com/nemgrouplimited/symbol-desktop-wallet/pull/892) submitted to the integration branch.

**Pull request closed without merging**